src/c_kzg_4844.c

```diff
@@ -1513,16 +1510,18 @@ C_KZG_RET verify_blob_kzg_proof_batch(
         );
         if (ret != C_KZG_OK) goto out;
 
+        Polynomial polynomial;
```

Comment:
Nitpick: I think we tend to declare variables in the beginning of each scope (that is, right after the `for` line in this case). No strong opinion from my side.

Reply:
Yeah, I agree with @asn-d6 here. Let's put that at the top of the loop, before `bytes_to_kzg_commitment`.

Reply:
No strong opinion here, did this mostly to convince myself that (re-)using the variable was safe (this way, further using it after the loop would not compile).

Reply:
Fixed.